include/oneapi/dpl/pstl/ranges/nanorange.hpp

```diff
@@ -15609,7 +15612,7 @@ inline constexpr bool is_raco<common_view_fn> = true;
 namespace views
 {
 
-NANO_INLINE_VAR(::nano::detail::common_view_fn, common)
+NANO_INLINE_VAR(nano::detail::common_view_fn, common)
```

Comment:
Why don't we use `__nanorange` here?

Reply:
because we are in `__nanorange` scope here.

```
NANO_INLINE_VAR(nano::detail::transform_view_fn, transform)
}

NANO_END_NAMESPACE
```